vuecomponents/tool/src/main/frontend/src/resources/utils.js

```diff
@@ -0,0 +1,5 @@
+export function replacer(key, value) {
```

Comment:
Is the purpose of mainly for the dynamic rubrics or something general?
In this case I would propose to change the file-path to something like: `./src/utils/dynamic-rubrics-utils.js`
In case it's something useful for all vue-webcomponents a comment explaining it would be nice and I think a path like this would be good: `./src/utils/core-utils.js`